.github/workflows/members.json

```diff
@@ -0,0 +1,15 @@
+[
```

Comment:
Can this also include a few folks from AION SDK for dealing with BYOID code changes? I think @lsirac @aeitzman

Reply:
Done.